framework/log/SyslogTarget.php

```diff
@@ -40,13 +46,22 @@ class SyslogTarget extends Target
         Logger::LEVEL_ERROR => LOG_ERR,
     ];
 
+    /**
+     * @inheritdoc
+     */
+    public function init()
+    {
+        if ($this->options === null) {
+            $this->options = LOG_ODELAY | LOG_PID;
+        }
+    }
```

Comment:
call to parent::init() is missing. I know it is empty right now but later code changes may add something to it.